Parkinson/Startup/APHAppDelegate.m

```diff
@@ -73,6 +73,11 @@ - (BOOL)application:(UIApplication*) __unused application willFinishLaunchingWit
 
     return YES;
 }
+- (BOOL)application:(UIApplication *) __unused application didFinishLaunchingWithOptions:(NSDictionary *) __unused launchOptions {
+    self.onboardingManager.showShareAppInOnboarding = YES;
+    
```

Comment:
Now that I look at this in context, it seems like it would make even more sense to do this just after the call to [super ...] in application:willFinishLaunchingWithOptions:. This is probably fine too though.